templates/fine-tune-llm_v2/cookbooks/optimize_cost/README.ipynb

```diff
@@ -0,0 +1,302 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tuning Configs for Cost or Performance\n",
+    "\n",
+    "**⏱️ Time to complete**: 30 minutes\n",
+    "\n",
+    "Making models train faster is directly related to reducing the cost. But that is not the only objective that you should optimize.\n",
+    "Some people may not have access to the latest and greatest Nvidia GPUs, but still want to fine-tune LLMs on cheaper GPUs like L4s or A10s. So within this limitation you also want to 1) unblock yourself (even if it means more $/flops) and 2) optimize for best configuration while maintaining the quality of the models.\n",
+    "\n",
+    "LLMForge is an internal library developed at Anyscale which unifies the fine-tuning experience across Anyscale products. This is the core framework that powers both the finetuning API and this template. In the template, the library enables you to also choose certain knobs to accelerate training of the LLM for your specific application. In this cookbook, we will discuss the parameters that are exposed that can help squeeze more flops out of GPUs and train models faster. "
```

Comment:
Lines about llmforge here are better suited in the main  .ipynb for the template